Rex/Misc/Sparrow/__module__.pm

```diff
@@ -0,0 +1,142 @@
+package Rex::Misc::Sparrow;
+
+use Rex -base => [qw(1.4 exec_autodie)];
+use Rex::CMDB;
+
+use Rex::Lang::Perl::Cpanm;
+
+use File::Spec;
+use YAML;
+
+our $sparrow = get cmdb 'sparrow';
+
+unless ( defined $sparrow ) {
+  my $yaml =
+    YAML::LoadFile( Rex::Helper::Path::get_file_path('files/sparrow.yml') );
+  $sparrow = $yaml->{sparrow};
+}
+
+our $sparrow_temp = File::Spec->tmpdir();
+
+=pod
+
+=head1 NAME
+
+Rex::Misc::Sparrow - manage and run sparrow checks
+
+=head1 DESCRIPTION
+
+Setup sparrow and its checks described in the configuration file, and run them.
+
+=head1 USAGE
+
+ $ rexify --use=Rex::Misc::Sparrow
+ $ rex Misc:Sparrow:setup
+
+=head1 TASKS
+
+=over 4
+
+=item setup
+
+Installs sparrow plugin via cpanm, and configures checks.
+
+=cut
+
+desc 'Setup sparrow';
+task 'setup', sub {
+  cpanm -install;
+  cpanm -install => [qw(Digest::MD5 Test::More Sparrow~0.0.21)];
+
+  needs 'configure';
+};
+
+=item configure
+
+Installs and configures sparrow checks as described in either CMDB or in C<files/sparrow.yml>.
+
+=cut
+
+desc 'Configure sparrow checks';
+task 'configure', sub {
+  run 'sparrow index update';
+
+  foreach my $project ( keys %{$sparrow} ) {
+    run "sparrow project create $project";
+
+    foreach my $check ( @{ $sparrow->{$project} } ) {
+      run "sparrow plg install $check->{plugin}";
+      run "sparrow check add $project $check->{checkname}";
+      run "sparrow check set $project $check->{checkname} $check->{plugin}";
+      configure_check( $project, $check );
+      run "sparrow check show $project $check->{checkname}";
```

Comment:
> I think I'd prefer removing it from configure task and having a separate dumpconfig task, which runs sparrow check show $project $check for all checks (and later it would also support partial dumps via CLI options).

Sounds good!

> The debug output already shows this if I need to debug ...

Yeah I know that debug shows it, the only little "hurdle" about using `-d` is too large / verbose output from it ... probably this is out of the scope of this task, but just want to note this here ... But dumpconfig will be fine ...